roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -20,7 +21,7 @@
         "/var/lib/libvirt:/var/lib/libvirt",
         "/run/libvirt:/run/libvirt:shared",
         "/var/lib/nova:/var/lib/nova:shared",
-        "/var/lib/iscsi:/var/lib/iscsi:z",
+        "/var/lib/iscsi:/var/lib/iscsi",
```

Comment:
ok so your removing :z so that we will not update the selinux lables to container_file_t exctra when the contianer starts.

that probaly correct and is in line with how we treat libvirt but you need to make sure the openstack selinux package set the approate rules to allwo the contaienr to interact with the socket.

assuming you have checked that then this is probaly fine.

Reply:
Yeah, iscsid won't run on the host if it doesn't like the context labels on those directories. You'll see a couple of tasks in the install playbook that ensures the proper context is restored if necessary. This is a critical piece to the upgrade process, where the labels where modifed by the previous ":z" container mounts.

Reply:
This isn't a one-off change, but there are tasks in my install code that I suspect can be moved to the update playbook, which I didn't know existed until now. I'll need to look at this further.